charts/incubator/privatebin/values.yaml

```
@@ -0,0 +1,60 @@
+image:
+  repository: privatebin/pdo
+  pullPolicy: IfNotPresent
+  tag: 1.4.0@sha256:5fe4b94286dcadbd0e576418f619205463030754e257863ec430eb7eb3e957fd
+
+env:
+  # See also: https://github.com/PrivateBin/docker-nginx-fpm-alpine#timezone-settings
+  TZ: "{{ .Values.TZ }}"
+  PHP_TZ: "{{ .Values.TZ }}"
+
+securityContext:
+  # This image supports running in read-only mode. Using this reduces the attack
+  # surface slightly, since an exploit in one of the images services can't
+  # overwrite arbitrary files in the container. Only /tmp, /var/tmp, /var/run &
+  # /srv/data may be written into.
+  # See also: https://github.com/PrivateBin/docker-nginx-fpm-alpine#running-the-image
+  readOnlyRootFilesystem: true
+  privileged: false
+  allowPrivilegeEscalation: false
+
+service:
+  main:
+    protocol: HTTP
+    ports:
+      main:
+        targetPort: 8080
+        port: 8080
```

```suggestion
        port: 10247
```
We try to keep ports used different for every app, so users won't experience conflicts by doing the default installs
The next available port can be found here. at the bottom
https://truecharts.org/manual/default-ports
(While it says 10246, it used. the next is 10247)
We will soon have the mentioned doc automatic generated. 
(You don't have to update it)